pay-api/src/pay_api/services/cfs_service.py

```diff
@@ -38,21 +38,23 @@ class CFSService(OAuthService):
     """Service to invoke CFS related operations."""
 
     @classmethod
-    def create_cfs_account(cls, name: str, contact_info: Dict[str, Any],
-                           payment_info: Dict[str, any] = None, receipt_method: str = None) -> Dict[str, str]:
+    def create_cfs_account(cls, identifier: str, contact_info: Dict[str, Any],  # pylint: disable=too-many-arguments
+                           payment_info: Dict[str, any] = None,
+                           receipt_method: str = None, site_name=None) -> Dict[str, str]:
         """Create a cfs account and return the details."""
-        name = re.sub(r'[^a-zA-Z0-9]+', ' ', name)
+        party_prefix = current_app.config.get('CFS_PARTY_PREFIX')
+        party_id = re.sub(r'[^a-zA-Z0-9]+', ' ', f'{party_prefix}{identifier}')  # TODO is the prefixing needed?
```

Comment:
I am not sure if this is really needed...do we need it?

Reply:
It was needed when we were using entity names (first version of CC) as the party name. No longer needed